### DISCUSSION TITLE
<FluentButton Target="_blank"> not opening a new browser tab

### DISCUSSION THREAD

**Discussion by andrewwang1tw** [microsoft/fluentui-blazor#2408] [Q&A] [answered]
I saw there is the parameter "Target" for "FluentButton", but how to use it?

I use below
  ```
<FluentButton @onclick="NavToAbout" BackgroundColor="rgba(0, 0, 0, 0)" Color="rgba(255, 255, 255, 255)" Target="_blank">About</FluentButton>  

@code {
    public void NavToAbout()
    {
        NavigationManager.NavigateTo("https://learn.microsoft.com/aspnet/core/");
    }
```

It did navigate to the specific URL but on current browser tab, not openning a new browser Tab ...

**Comment by vnbaaij** [comment, marked as answer]
The `Target` parameter is used to set the html's button `formtarget`. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#formtarget for more information.

It is not exactly the same as athe target for an anchor.

**Reply by andrewwang1tw** [reply]
Thanks.
